examples/sampling/graphbolt/rgcn/hetero_rgcn.py

```diff
@@ -420,7 +421,7 @@ def print_statistics(self, run=None):
             r = best_result[:, 2]
             print(f"  Final Train: {r.mean():.2f} ± {r.std():.2f}")
             r = best_result[:, 3]
-            print(f"   Final Test: {r.mean():.2f} ± {r.std():.2f}")
+            print(f"Test accuracy {r.mean():.4f}")
```

Comment:
`--runs` could be removed, then we could omit `r.std()` for all fields. that makes more consistent. combine this PR with https://github.com/dmlc/dgl/pull/6808? please reach out MIngbang

Reply:
OK